cloud.foundry.cli/src/main/java/cloud/foundry/cli/crosscutting/mapping/CfArgumentsCreator.java

```diff
@@ -0,0 +1,92 @@
+package cloud.foundry.cli.crosscutting.mapping;
+
+import picocli.CommandLine;
+import picocli.CommandLine.ParseResult;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Properties;
+
+import cloud.foundry.cli.crosscutting.logging.Log;
+
+/**
+ * This utility class creates certain options with default values, which were not passed when the command was called.
+ * There are currently default values for <code>api</code>, <code>organization</code> and <code>space</code>.
+ */
+public class CfArgumentsCreator {
+
+    /**
+     * Name of the property file, which contains default values
+     */
+    private static final String CF_CONTROL_PROPERTIES = "cf_control.properties";
```

Comment:
Just a minor thing, but the project is called `cf-control` with a dash everywhere; I'd prefer to be consistent about that.

Reply:
Done.